.circleci/os/linux/dependencies.sh

```
@@ -19,4 +19,9 @@ export PATH="$PATH:/home/qa/.yarn/bin"
 npx playwright install
 
 # verify env var
-echo $PUPPETEER_EXECUTABLE_PATH
\ No newline at end of file
+echo $PUPPETEER_EXECUTABLE_PATH
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)